protocol-configuration/mainnetAddresses.ts

```diff
@@ -2038,11 +2038,16 @@ const MainnetContractsConfig: MainnetContractsConfig = {
     address: '0x84dc71500D504163A87756dB6368CC8bB654592f',
     category: AddressCategory.Volt
   },
-  podExecutor: {
+  oldPodExecutor: {
     artifactName: 'PodExecutor',
     address: '0x99d8b669F48A708f7C0373AF6CE31F0726ab6CaD',
-    category: AddressCategory.Governance
+    category: AddressCategory.Deprecated
   },
+  // podExecutor: {
```

Comment:
TODO: Fill in once deployed

Reply:
I really don't like changing the names of contracts in this file to upgrade to new ones (or deprecate old ones). Anyone that reads from this file (like Storm) will probably be assuming that entries don't change except for their category (if deprecated).

Can we use podExecutorV2 as the name instead?